package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "compile": "buidler compile",
     "test": "buidler test",
-    "coverage": "buidler coverage",
+    "coverage": "IS_COVERAGE='true' buidler coverage --network buidlerevm",
```

Comment:
Here I add this environment variable to not fully test results of transactions when it is doing coverage tests.